frontends/mit-open/src/pages/HomePage/HomePage.tsx

```diff
@@ -28,8 +28,10 @@ const FeaturedCoursesCarousel = styled(ResourceCarousel)(({ theme }) => ({
 }))
 const MediaCarousel = styled(ResourceCarousel)(({ theme }) => ({
   margin: "80px 0",
+  minHeight: "388px",
```

Comment:
This preserves the height of the section as it was collapsing as I tabbed through the media types, though we may want to do this in the carousel itself.